src/main/kotlin/br/com/bumblebee/configuration/feign/election/ElectionConfiguration.kt

```diff
@@ -3,7 +3,9 @@ package br.com.bumblebee.configuration.feign.election
 import br.com.bumblebee.configuration.feign.FeignConfiguration
 import br.com.bumblebee.configuration.feign.election.decoder.ElectionDecoder
 import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
 
+@Configuration
```

Comment:
https://cloud.spring.io/spring-cloud-netflix/multi/multi_spring-cloud-feign.html#spring-cloud-feign-overriding-defaults

Reply:
`FooConfiguration` does not need to be annotated with `@Configuration`. However, if it is, then take care to exclude it from any `@ComponentScan` that would otherwise include this configuration as it will become the default source for `feign.Decoder`, `feign.Encoder`, `feign.Contract`, etc., when specified. This can be avoided by putting it in a separate, non-overlapping package from any `@ComponentScan` or `@SpringBootApplication`, or it can be explicitly excluded in `@ComponentScan`.

Nāo entendo 🤓